### DISCUSSION TITLE
Librewolf Doesn't Like The Latest Release

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [onemen/TabMixPlus#222] [General]
Claims the addon is corrupt and won't install it

**Comment by onemen** [comment]
can you provide a link to Librewolf, and steps to reproduce this issue?

**Reply by neurodiverseEsoteric** [reply]
librewolf.net

The steps are the same for installing any addon as they are for firefox....

**Comment by onemen** [comment]
Read our [Installation](https://github.com/onemen/TabMixPlus#installation) instructions
